investigate reported t/zz-plugin-cycle.t error from cpantesters

more details from http://www.cpantesters.org/cpan/report/545b51f7-6c24-1014-970c-d94e51739abc
using Template-Toolkit-3.009
```
plugin error - Can't locate object method "load" via package "test::Simple" (perhaps you forgot to load "test::Simple"?) at lib/Template/Plugins.pm line 217.
# Looks like your test exited with 255 before it could output anything.
t/zz-plugin-cycle.t .......... 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 1/1 subtests 
t/zz-plugin-leak-gh-213.t .... skipped: Broken on older perls. We need to sort this out once everything is passing
t/zz-plugin-leak-rt-46691.t .. ok
t/zz-plugin-leak.t ........... skipped: Internal test for developer, add the --dev flag to run
t/zz-pmv.t ................... skipped: These aren't the tests you're looking for... move along
t/zz-pod-coverage.t .......... skipped: Author tests not required for installation
t/zz-pod-kwalitee.t .......... skipped: Author tests not required for installation
t/zz-stash-xs-leak.t ......... skipped: Internal test for developer, add the --dev flag to run
t/zz-url2.t .................. skipped: We can't agree on the right joint for the URL plugin

Test Summary Report
-------------------
t/zz-plugin-cycle.t        (Wstat: 65280 Tests: 0 Failed: 0)
  Non-zero exit status: 255
  Parse errors: Bad plan.  You planned 1 tests but ran 0.
Files=103, Tests=2924, 32 wallclock secs ( 0.08 usr  0.01 sys + 10.62 cusr  7.47 csys = 18.18 CPU)
Result: FAIL
Failed 1/103 test programs. 0/2924 subtests failed.
make: *** [Makefile:1228: test_dynamic] Error 255
```

--- a/t/zz-plugin-cycle.t
+++ b/t/zz-plugin-cycle.t
@@ -23,15 +23,19 @@ use Test::More;
 
 plan skip_all => "Developer test" unless ( $ENV{AUTOMATED_TESTING} or $ENV{RELEASE_TESTING} );
 
-#use Test::LeakTrace;
 eval { require Test::LeakTrace };
 if ( $@ or !$INC{'Test/LeakTrace.pm'} ) {
     plan skip_all => 'Test::LeakTrace not installed';
 }
 
-plan tests => 1;
+plan tests => 2;
+
+note "plugin_simple_test();";
+
+ok plugin_simple_test(), "plugin_simple_test";
 
 note "Searching for leak using Test::LeakTrace...";
+
 
 my $no_leaks = Test::LeakTrace::no_leaks_ok( \&plugin_simple_test, 'no leak from Template::Plugin' );
 
@@ -49,9 +53,25 @@ if ( !$no_leaks ) {
 exit;
 
 sub plugin_simple_test {
-    my $tpl = Template->new( PLUGIN_BASE => 'test' );
+    my $tpl = Template->new({
+        PLUGIN_BASE => [ 'test' ],
+        DEBUG => 1,
+    }) or die;
     $tpl->context->plugin( 'Simple', [] );
 
     return $tpl;
 }
 
+package test::Simple;
+
+sub new {
+    my ($pkg) = @_;
+    return bless {}, $pkg;
+}
+
+sub load {
+    my $class   = shift;
+    return $class;
+}
+
+1;

